Shared/Samples/Project geometry/ProjectGeometryView.swift

```diff
@@ -39,20 +43,20 @@ struct ProjectGeometryView: View {
                     model.pointGraphic.geometry = projectedPoint
                     
                     // Updates the location callout placement.
-                    calloutPlacement = LocationCalloutPlacement(location: projectedPoint)
+                    calloutPlacement = CalloutPlacement.location(projectedPoint)
                 } else {
                     // Hides the callout and point graphic.
                     calloutPlacement = nil
                     model.pointGraphic.geometry = nil
                 }
             }
-            .callout(placement: $calloutPlacement.animation(.default.speed(2))) { callout in
+            .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
                 VStack(alignment: .leading) {
                     Group {
                         Text("Coordinates")
                             .fontWeight(.medium)
-                        Text("Original: \(originalPoint!.xyCoordinates)")
-                        Text("Projected: \(callout.location.xyCoordinates)")
+                        Text("Original: \(originalPoint.xyCoordinates)")
+                        Text("Projected: \(tapLocation.xyCoordinates)")
```

Comment:
This is a mistake that causes the coordinates displayed in the callout to be the same.